### PR TITLE
Cleanup setup.py - install_requires

### PR DESCRIPTION
**Pull request by JshWright**
Per the Python packaging guidelines[1], setup.py should only contain the minimal
requirements needed to run the code correctly. This commit removes the dev and
test dependencies from setup.py, which should not be installed in production envs.

[1]: https://packaging.python.org/discussions/install-requires-vs-requirements/

As this commit contains no functional changes to the okta-sdk-python library, I consider this an "obvious fix" for the purposes of the contributor guidelines, and therefore a CLA is not necessary (I'm happy to sign one if the maintainer feels differently though)

### CODE REVIEW

**Comment by bretterer** [comment]
Hey @JshWright!  Thank you for this PR. I need to pull this down and make sure we don't have any issues here, so it may be a few days before we can get this merged in.

I believe this is an obvious fix as well, but since it does change functionality and you don't mind, go ahead and sign and send a CLA over to us.

Ill keep you updated on this PR

File: setup.py
Comment (JshWright) [reply setup.py:38]: Yeah, that's a good call out. I only removed the really obvious testing related libraries, but that's likely not needed either.

Comment (serhiibuniak-okta) [reply setup.py:38]: Yes, looks like this change should resolve this issue - https://github.com/okta/okta-sdk-python/issues/157

**Review by bretterer — APPROVED** [review]

**Comment by serhiibuniak-okta** [comment]
Hi @JshWright. I've included your commit along with another one related into this https://github.com/okta/okta-sdk-python/pull/159 and just merged to master. Need to wait until we release v1.2.2. Thanks for your contribution.

**Comment by JshWright** [comment]
Thanks @serhiibuniak-okta, looking forward to 1.2.2!

**Comment by serhiibuniak-okta** [comment]
@JshWright we decided to include some additional changes, so we released v1.3.0. It is ready. Thanks again.

**Pull request closed without merging**
